charts/sonarqube/templates/apply-aml-github-job.yaml

```diff
@@ -0,0 +1,45 @@
+{{- if .Values.aml }}
+{{- if .Values.aml.github }}
+{{- if and .Values.aml.github.appId .Values.aml.github.url .Values.aml.github.clientId .Values.aml.github.clientSecret .Values.aml.github.key .Values.aml.github.privateKey }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "sonarqube.fullname" . }}-apply-aml-github-hook
+  labels:
+    app: {{ template "sonarqube.name" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade
```

Comment:
i think this would do harm if ran after upgrade. The DevOpsPlatform settings (previously known as ALM) persist in the database and do not need to be re added after a upgrade

Reply:
I decided to add on post-install and post-upgrade because this way these values can be changed using a pipeline or something like that without a fresh install.